_includes/nav.html

```diff
@@ -53,7 +53,7 @@
         <div class="collapsible-body">
           <ul>
             <li class="waves-effect {% if page.permalink contains 'guidelines' %}active{% endif %}">
-              <a href="/guidelines/">Guidelines</a>
+              <a href="/guidelines/">Guidelines &amp; FAQ</a>
```

Comment:
```suggestion
              <a href="/guidelines/" class="overflow">Guidelines &amp; FAQ</a>
```

need to an an `overflow` class to this `<a>` to get its text to wrappp

Reply:
changes made!